### PR TITLE
chore(ci): convert workflows to use slab action

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: .github/workflows/wasm_client_benchmark.yml [hunk 1]
```diff
@@ -1,62 +1,93 @@
...
+        with:
+          since_last_remote_commit: true
+          files_yaml: |
+            wasm_bench:
+              - tfhe/Cargo.toml
+              - concrete-csprng/**
+              - tfhe-zk-pok/**
+              - tfhe/src/**
+              - '!tfhe/src/c_api/**'
+              - .github/workflows/wasm_client_benchmark.yml
```
Comment (IceTDrinker) [inline .github/workflows/wasm_client_benchmark.yml:49]: tfhe/web_wasm_parallel_tests as well ?

File: ci/slab.toml [hunk 2]
```diff
@@ -31,31 +31,6 @@ environment_name = "canada"
...
-workflow = "aws_tfhe_signed_integer_tests.yml"
-profile = "cpu-big"
-check_run_name = "CPU Signed Integer AWS Tests"
-
-[command.cpu_wasm_test]
-workflow = "aws_tfhe_wasm_tests.yml"
-profile = "cpu-small"
-check_run_name = "CPU AWS WASM Tests"
-
-[command.gpu_test]
```
Comment (IceTDrinker) [inline ci/slab.toml:54]: outdated commands ?

File: ci/slab.toml [hunk 3]
```diff
@@ -120,23 +80,3 @@ check_run_name = "Boolean CPU AWS Benchmarks"
 workflow = "core_crypto_benchmark.yml"
 profile = "bench"
 check_run_name = "Core crypto CPU AWS Benchmarks"
-
```
Comment (IceTDrinker) [inline ci/slab.toml:123]: all commands outdated ?

Comment (soonum) [reply ci/slab.toml:123]: Yes

File: ci/slab.toml (hunk 2 above)
Comment (soonum) [reply ci/slab.toml:54]: Yes indeed.

**Review by IceTDrinker — APPROVED** [review]

**Pull request merged**